app/models/miq_widget_set.rb

```diff
@@ -41,14 +46,14 @@ def update_members
   end
 
   def add_to_dashboard_order
-    return unless group
+    return if userid || group.nil? # only for group version
```

Comment:
Not sure why tests have widget sets without a group. they should be required for all but the read-only dashboards.

Not sure if these tests are from before we made group required.

Reply:
I'm not sure the premise is true.  Widget Sets can belong to a group or a user, so it's not surprising to me that you can have one and not the other.

Reply:
But when the owner is a user, it still was associated with a group. (the user's group) and the code still needed to access the group. So having the code just use the group association made sense.